submissions/yaripey/dom-task/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/main.css">
+  <script src="js/main.js"></script>
```

Comment:
Why did you place this script inside the `head` tag?

Reply:
Moved it to the bottom.